snippets/fahrenheit_to_celsius.md

```diff
@@ -0,0 +1,19 @@
+---
+title: fahrenheit_to_celsius
+tags: conversion, beginner
+---
+
+Converts Fahrenheit to Celsius.
+
+celsius = (fahrenheit - 32) / 1.8
+
+```py
+def fahrenheit_to_celsius(fahrenheit):
+  return ((fahrenheit - 32) / 1.8)
+```
+
+```py
+fahrenheit_to_celsius(77) # 25.0

```

Comment:
No need for that many examples, one will suffice as it's a straightforward use-case.